src/OpenTelemetry/Instrumentation/DiagnosticSourceSubscriber.cs

```diff
@@ -81,6 +81,9 @@ public void OnError(Exception error)
         {
         }
 
+        /// <summary>
```

Comment:
@eddynaka FYI when implementing an interface or something on the base class you can use:
```csharp
/// <inheritdoc/>
```
That way you don't need to reproduce the same text all over.

Reply:
Looks like you are using `<inheritdoc/>` a bunch below just not on `Dispose`.

Reply:
i will check. i think VS wasn't suggesting it. it's why i created. let me double-check and update